### PR TITLE
fix(pipeline-modal): fix table row width and content for pipeline modal

### PR DESCRIPTION
**Pull request by rohitkrai03**
## Fixes 

- https://issues.redhat.com/browse/KFLUXBUGS-1127

## Description

- Fix the way content inside the table behaved by adding proper modifier.
- Added a truncate option for image url.

## Type of change

- [x] Bugfix

## Screen shots / Gifs for design review 

Screenshot 2024-04-09 at 5 31 32 PM

## Browser conformance: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: src/components/CustomizedPipeline/CustomizePipelinesModal.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const createCustomizeAllPipelinesModalLauncher = (
 ) =>
   createRawModalLauncher(CustomizeAllPipelines, {
     'data-testid': 'customized-all-pipelines-modal',
-    variant: ModalVariant.large,
+    width: '75%',
```
Comment (abhinandan13jan) [inline src/components/CustomizedPipeline/CustomizePipelinesModal.tsx:15]: Just a suggestion. Fixed width is still causing problems is smaller screens. I guess using <Truncate> in external link text it is being displayed properly.
```
<ExternalLink
                href={
                  component.spec.containerImage.startsWith('http')
                    ? component.spec.containerImage
                    : `https://${component.spec.containerImage}`
                }
                text={<Truncate content={component.spec.containerImage} />}
              />
              ```

Comment (abhinandan13jan) [reply src/components/CustomizedPipeline/CustomizePipelinesModal.tsx:15]: Issue still in smaller screen sizes
Screenshot 2024-04-18 at 5 47 59 PM

Using Truncate on External link text
Screenshot 2024-04-18 at 5 56 14 PM

Comment (rohitkrai03) [reply src/components/CustomizedPipeline/CustomizePipelinesModal.tsx]: Fixed it with better handling of table row width - 

https://github.com/openshift/hac-dev/assets/6041994/195d03ba-345b-4562-a866-9fd12c3be1a3

**Review by sahil143 — APPROVED** [review]
/lgtm

**Pull request merged**